tests/acceptance/features/apiSpacesShares/shareSpaces.feature

```diff
@@ -42,6 +42,15 @@ Feature: Share spaces
       | root@@@permissions@@@1@@@roles@@@0                           | viewer    |
 
 
+  Scenario: A user can see that the group has been granted access
+    Given group "sales" has been created
+    And user "Alice" has shared a space "share space" to group "sales" with role "viewer"
```

Comment:
can this step be converted to `When` and also checking https or ocs status steps would be better?